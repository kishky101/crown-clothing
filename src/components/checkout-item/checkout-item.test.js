import renderer from 'react-test-renderer';
import CheckoutItem from './checkout-item.component';
import { MockStore } from '../../utils/test/setup';

test('creating checkout-item snapshot', () => {
    const checkoutitem = renderer.create(
        <MockStore>
            <CheckoutItem product= {{
                id: 6,
                name: 'Palm Tree Cap',
                imageUrl: 'https://i.ibb.co/rKBDvJX/palm-tree-cap.png',
                price: 14,
                quantity: 2
            }} />
        </MockStore>
    ).toJSON
    expect(checkoutitem).toMatchSnapshot();
})