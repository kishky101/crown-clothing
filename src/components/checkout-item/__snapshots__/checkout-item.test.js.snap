// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creating checkout-item snapshot 1`] = `[Function]`;
